src/pages/Login.jsx

```diff
@@ -1,10 +1,7 @@
-//smart component
 import React from "react";
-import LoginForm from "@features/authentication/components/LoginForm/LoginForm";
-import { useAuth } from "@features/authentication/hooks/useAuth";
+import LoginForm from "@features/authentication/components/LoginForm";
 const Login = () => {
-  const { login } = useAuth();
-  return <LoginForm onLogin={login} />;
+  return <LoginForm />;
```

Comment:
we should also have logic here that if you are logged in you cannot come back here.
You need to go back to the root